src/com/jambit/onboarding2020/tbrpg/core/GameInput.java

```diff
@@ -36,6 +36,21 @@ public int inputInteger() throws InvalidInputException{
         return intInput;
     }
 
+    public String inputStringToUppercase() throws InvalidInputException {
```

Comment:
this MethodName is kinda missleading, since I expected from the name, that it will turn the inputString to uppercase. Instead, it reads the input, does some exception handling and finally it converts that input to uppercase. Maybe a better name would be: getUserInput()? :)